### PR TITLE
Reorganize structure

### PR DESCRIPTION
**Pull request by AntonLee**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -71,61 +65,48 @@ gulp.task('makeDemo', function () {
...
+gulp.task('js', function () {
     return gulp.src(jsDist)
-        .pipe(sourcemaps.init())
-        .pipe(concat({path: 'bundle.min.js', cwd: ''}))
-        .pipe(uglify())
-        .pipe(gulp.dest(path.dist.js))
-        .pipe(rev())
-        .pipe(sourcemaps.write('.'))
+        .pipe(concat({path: 'lib.min.js', cwd: ''}))
+        //.pipe(uglify())
```
Comment (Rod-rig) [inline gulpfile.js]: please returrn back uglifying

Comment (AntonLee) [reply gulpfile.js]: done

**Review by Rod-rig — COMMENTED** [review]
1. error message in header (Undefined index: MODIFIER in markup\components\nav\menu.php on line 7)
2. Please remove all unnecessary images and styles
3. static page is unuseful

**Comment by AntonLee** [comment]
1. there is two ways of resolving this: add an empty parameter "MODIFIER", or implementing default parameters
2. do you have something specific in mind?
3. I've added it on purpose. To show that static pages should not contain any specific markup like .container

**Comment by AntonLee** [comment]
`includeComponent('nav/menu', 'menu/simple', ['MODIFIER' => ''])`
looks a bit ugly compared to 
`includeComponent('nav/menu', 'menu/simple')`

it also means if you're adding a new parameter you'll have to update all of component's inclusions

but on other hand, it's simple

**Comment by AntonLee** [comment]
I'll implement default parameters, remove sample images and styles and document it.
But I think static page should stay.

**Comment by Rod-rig** [comment]
ok

**Review by Rod-rig — COMMENTED** [review]
1. can you remove favicon link on sitemap page?
2. please fix 404 error for local.js

**Comment by AntonLee** [comment]
oops, didn't mean to delete `local.js`.

**Comment by Rod-rig** [comment]
Good job @AntonLee :clap:

**Pull request merged**